### PR TITLE
Response

### PR DESCRIPTION
**Pull request by Anddrew914**
comfort 3
clarity 3, still some confusion when things don't work like I think they will

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -54,8 +52,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
+absolute
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:55]: Correct, but it asks you to explain: the first `/` represents the root of the system.

File: diagnostic.md [hunk 2]
```diff
@@ -69,13 +66,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 ```
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
-
- <!-- Answer Starts Here -->
+relative
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:69]: Correct, but it asks you to explain your answer: `planning` is referenced with respect to the current working directory, `project`.

File: diagnostic.md [hunk 3]
```diff
@@ -69,13 +66,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
...
+ <!-- Answer Starrelativets Here -->
 
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
-
+relative
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:75]: Correct, but it asks you to explain your answer: the image file lives inside our current working directory, project; as a result, an absolute path is not needed to access it.

**Comment by Jcornmanhomonoff** [comment]
7/7! Awesome job! Keep up the fantastic work! 🦄 👍

File: diagnostic.md (hunk 3 above)
Comment (Anddrew914) [reply diagnostic.md:75]: Apologies, I was running out of time so I felt like answering all of them quickly was better than not at all. Thanks!

**Pull request closed without merging**